logging/subscriber/src/subscriber.ts

```diff
@@ -0,0 +1,34 @@
+import type { EventPayload, Layer } from '@rango-dev/logging-types';
+
+import { EventType, Level } from '@rango-dev/logging-types';
+
+import { isEnabled } from './helpers';
+
+interface Options {
+  maxLevel: Level;
+}
+
+const DEFAULT_LEVEL = Level.Trace;
+
+export function init(layers: Layer[], options: Options) {
+  const { maxLevel } = options;
+
+  document.addEventListener(EventType, function handler(e) {
+    // Typescript issue: https://github.com/microsoft/TypeScript/issues/28357
+    const event = e as CustomEvent<EventPayload>;
+    const level = event.detail.level || DEFAULT_LEVEL;
+
+    if (!isEnabled(maxLevel, level)) {
+      // Skipping event.
+      return;
+    }
+
+    for (const layer of layers) {
+      try {
+        layer.handler(event.detail);
+      } catch {
+        break;
```

Comment:
we can use **continue** instead of **break**, Because if a layer encounters an error
The other layer can continue to work.

Reply:
It's intentional. it shouldn't proceed.